### PR TITLE
Add pcg package

### PR DESCRIPTION
**Pull request by isaachier**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/isaachier/hunter/build/1.0.393
  * https://travis-ci.org/isaachier/hunter/builds/353352987

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull request https://github.com/ingenue/hunter/pull/210

### CODE REVIEW

File: cmake/projects/pcg/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME pcg
+    VERSION "0.0.0"
```
[the remark this answers is not in the document]
Comment (isaachier) [reply cmake/projects/pcg/hunter.cmake]: Interesting. Will update.

File: examples/pcg/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.1)
+
+include("../common.cmake")
+
+set(CMAKE_C_STANDARD 99)
```
[the remark this answers is not in the document]
Comment (isaachier) [reply examples/pcg/CMakeLists.txt]: Got it. Good summary!

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3459
* https://travis-ci.org/ingenue/hunter/builds/353433345?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3460
* https://travis-ci.org/ingenue/hunter/builds/353455693?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.20.20

**Pull request merged**